chainweb.cabal

```diff
@@ -245,6 +248,9 @@ library
         , data-dword >= 0.3
         , deepseq >= 1.4
         , directory >= 1.3
+        -- add version bounds for direct-sqlite
+        , direct-sqlite
+        , ekg >= 0.4
```

Comment:
why?

Reply:
I'm not sure how that got there, I don't remember adding it. It has been removed. Chainweb still builds.